serverless.ts

```diff
@@ -24,8 +25,6 @@ const config: AWS = {
       SLACK_BOT_TOKEN: `${process.env.SLACK_BOT_TOKEN}`,
       HR_BOT_USER_ID: `${process.env.HR_BOT_USER_ID}`,
       MORE_TREES_API_KEY: `${process.env.MORE_TREES_API_KEY}`,
-      IAM_ACCESS_KEY_ID: `${process.env.IAM_ACCESS_KEY_ID}`,
```

Comment:
https://github.com/amido/treeumph/blob/dc1f592f3e50ef3f3adb546556eae99b584b414b/src/functions/slackbot/handler.ts#L51

Will still need the IAM key id and role of hackternoon